app/assets/javascripts/navigation.coffee

```diff
@@ -1,5 +1,12 @@
 ready = ->
-  $('.js-navigation').on('click', -> $('.loading-indicator').show())
+  $('.js-navigation').on('click', (event) ->
+    if (event.ctrlKey or # New tab (windows)
+    event.shiftKey or # New window
+    event.metaKey or # New tab (mac)
+    (event.button and event.button == 1) # New tab (middle click)
+    )
+      return
+    return $('.loading-indicator').show())
```

Comment:
So this is a good start, and I like the documentation, but what do you think about something like this?

I think extracting it out into it's own method makes it a little more readable.

Just a style note I'm a fan of guard clauses

```coffee
return if foo
# continue on
```

Also coffeescript much like Ruby the last line of the method is returned as the result. So the last result is not necessary.

```coffee
# A new window/tab event can be one of the following
# ctrlKey (Windows)
# metaKey (OSX)
# shiftKey (OS agnositc)
# New tab with middle click
new_window_or_tab_event = (event) ->
  middle_click_event = (event.button && event.button == 1)
  event.ctrlKey || event.metaKey || event.shiftKey || middle_click_event

ready = ->
  $('.js-navigation').on('click', (event) ->
    return if new_window_or_tab_event(event)
    $('.loading-indicator').show())
```